### ISSUE TITLE
Issue response should return CommentsUrl

### ISSUE THREAD

**Issue by shiftkey** [octokit/octokit.net#883]
https://developer.github.com/v3/issues/#list-issues

```json
{
    "id": 1,
    "url": "https://api.github.com/repos/octocat/Hello-World/issues/1347",
    "labels_url": "https://api.github.com/repos/octocat/Hello-World/issues/1347/labels{/name}",
    "comments_url": "https://api.github.com/repos/octocat/Hello-World/issues/1347/comments",
    "events_url": "https://api.github.com/repos/octocat/Hello-World/issues/1347/events",
    "html_url": "https://github.com/octocat/Hello-World/issues/1347",
    "number": 1347,
    "state": "open",
    ...
}
```

See #882 for related discussion.

**Comment by naveensrinivasan** [comment]
Can I take this one?

**Comment by Haacked** [comment]
@naveensrinivasan go for it!

**Comment by Haacked** [comment]
I assigned myself so nobody takes it. But go ahead and work on it @naveensrinivasan

**Comment by M-Zuber** [comment]
This is awesome. Let's me throw out ~200 loc and tests!

**Comment by shiftkey** [comment]
Fixed in #884
